nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -1 +1,141 @@
 # Upgrading from Nautobot v1.X
+
+## Updating from Nautobot 1.5 to 2.0
+
+### Enhanced Filter Fields
+
+Below is a table documenting [enhanced filter field changes](../release-notes/version-2.0.md#enhanced-filter-fields-2804) in v2.x.
+
+| Model                 | Enhanced Filter Field| Changes                                                  | UI and Rest API endpoints Available in v2.X|
+|-----------------------|----------------------|------------------------------------------------------------|----------------------------------------------|
+| ConsolePort           | `device`             | Enhanced to support primary key UUIDs in addition to names | `/dcim/console-ports/?device=<uuid|name>`|
+|                       | `cabled`             | Renamed to `has_cable`                                     | `/dcim/console-ports/?has_cable=True|False`|
```

Comment:
I wonder if we should split 'renamed' fields into a separate table from 'enhanced' fields?